lib/karaokium_web/live/admin/karaoke_live/show.ex

```diff
@@ -43,30 +41,6 @@ defmodule KaraokiumWeb.Admin.KaraokeLive.Show do
     {:noreply, reload(socket)}
   end
 
-  @impl true
-  def handle_event("open_voting", %{"id" => id}, socket) do
```

Comment:
Where are you moving this functionality to?

Reply:
Forgot to say, I will look to get_fsm and I will try no make something with it

Reply:
You can leave the `gen_fsm` for another PR :pray: